### ISSUE TITLE
[Feature Request] Codesigning on macOS

### ISSUE THREAD

**Issue by mavethee** [Vita3K/Vita3K#2184]
Since Apple is kind enough with `macOS on ARM requires applications to be signed.` I wonder, do you have any plans for codesigning the app on macOS?

**Comment by Naville** [comment]
Don't think this is enforced with x64 apps with rosetta

**Comment by Naville** [comment]
Plus, on AppleSilicon you can always sign locally

**Comment by mavethee** [comment]
> Don't think this is enforced with x64 apps with rosetta
> Plus, on AppleSilicon you can always sign locally

Yes, I'm aware, I just wanted to add Vita3K to Homebrew Casks and thats one of the requirements with x64 apps on ARM. If you don't want to have any other way for Mac users to install this emu that Vita3K's GitHub or webpage that's okay, just wondering if you're interested.

**Comment by Naville** [comment]
I'm not the author, just some random guy stumbling across the project. Not sure if the original developer is willing to pay 99$ and an entire new release process for this tbh.

I guess we're stick with signing it ourselves
